### PR TITLE
Feature/update example

### PR DESCRIPTION
**Pull request by LesTR**

### CODE REVIEW

File: example/tools/pom.xml [hunk 1]
```diff
@@ -31,6 +31,13 @@
      
   <name>${project.groupId}:${project.artifactId}</name>
 
+  <properties>
+    <apache.beam.groupId>cz.o2.proxima.beam</apache.beam.groupId>
+    <apache.beam.version>${project.version}</apache.beam.version>
+    <apache.beam.version.raw>2.18.0-SNAPSHOT</apache.beam.version.raw>
```
Comment (je-ik) [inline example/tools/pom.xml]: I think this should be inherited from parent. We can move this from `beam/pom.xml` to root `pom.xml`.

**Comment by dmvk** [comment]
@LesTR ping

File: example/server/pom.xml [hunk 2]
```diff
@@ -52,12 +52,6 @@
             <transformer
                 implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
           </transformers>
-          <relocations>
-            <relocation>
-              <pattern>com.google.common.</pattern>
-              <shadedPattern>${coreShade}.com.google.common.</shadedPattern>
```
[the remark this answers is not in the document]
Comment (LesTR) [reply example/server/pom.xml]: Returned back, this shading is still not nice, we should have probably clear vendor artifact for guava.

File: example/tools/pom.xml (hunk 1 above)
Comment (LesTR) [reply example/tools/pom.xml]: Moved.

**Comment by LesTR** [comment]
PR updated on tom of master + fixed some issues. Works now.

**Comment by je-ik** [comment]
:+1:

**Pull request merged**